src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -553,6 +558,26 @@ const MutationCollapsible = ({
                                   onResolveComment={index => handleResolveComment(`${therapyFirebasePath}/name_comments/${index}`)}
                                   onUnresolveComment={index => handleUnresolveComment(`${therapyFirebasePath}/name_comments/${index}`)}
                                 />
+                                <RCTButton
+                                  tooltipProps={
+                                    cancerTypeContainsSpecialCancerType || treatment.level === 'None'
+                                      ? {
+                                          overlay: cancerTypeContainsSpecialCancerType
+                                            ? tooManyRCTsText
+                                            : 'Add a highest level of evidence to view RCTs',
+                                        }
+                                      : null
+                                  }
+                                  disabled={treatment.level === 'None' || cancerTypeContainsSpecialCancerType}
```

Comment:
The logic of how the tooltip content rendered and when it's disabled are duplicated twice in below. Let's consolidate.